### PR TITLE
Add ~40 GitHub Administration questions

### PR DESCRIPTION
**Pull request by garysassano**
## PR Type

- [x] Adding new question(s)
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [ ] Documentation changes
- [ ] Other

## Related issue(s)

https://github.com/FidelusAleksander/ghcertified/issues/89

### CODE REVIEW

**Comment by FidelusAleksander** [comment]
Damn. Well done @garysassano , you're a champion :rocket: 

I will do my best to review this asap. 

Also if anyone studying for GitHub Administration exam wants to review this, leave thumbs up (as I am not experienced in this area)

File: content/questions/admin/question-044.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+question: "Which GitHub plans allow organizations to own an unlimited number of public and private repositories with a full feature set on private repositories? (choose two.)"
```
Comment (FidelusAleksander) [inline content/questions/admin/question-044.md:2]: What is "full feature set" ? 

GitHub Team for example does not include GitHub Advanced Security features for private repositories.

Make this more clear please + a link to what this refers to

File: content/questions/admin/question-042.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+question: "Which role in a GitHub organization has the authority to manage access to the organization's resources?"
+archetype: "questions"
+title: "Question 042"
+draft: false
+---
+
+1. [x] Organization owner
```
Comment (FidelusAleksander) [inline content/questions/admin/question-042.md:8]:
```suggestion
---

> https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization
1. [x] Organization owner
```

File: content/questions/admin/question-041.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+question: "What is the primary purpose of a GitHub organization?"
+archetype: "questions"
+title: "Question 041"
+draft: false
+---
+
+1. [x] To allow businesses and open-source projects to collaborate across many projects at once with advanced security and administrative features.
```
Comment (FidelusAleksander) [inline content/questions/admin/question-041.md:8]:
```suggestion
---

> https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations#about-organizations
1. [x] To allow businesses and open-source projects to collaborate across many projects at once with advanced security and administrative features.
```

File: content/questions/admin/question-043.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+question: "How can access management and collaboration be simplified within a GitHub organization?"
+archetype: "questions"
+title: "Question 043"
+draft: false
+---
+
+1. [x] By creating nested teams that reflect the group's structure, with cascading access permissions and mentions.
```
Comment (FidelusAleksander) [inline content/questions/admin/question-043.md:8]:
```suggestion
---

> https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations#about-organizations
1. [x] By creating nested teams that reflect the group's structure, with cascading access permissions and mentions.
```

File: content/questions/admin/question-045.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+question: "What is the advantage of having an enterprise account on GitHub Enterprise Cloud for an organization?"
+archetype: "questions"
+title: "Question 045"
+draft: false
+---
+
+1. [x] It allows owners to centrally manage policy and billing for multiple organizations.
```
Comment (FidelusAleksander) [inline content/questions/admin/question-045.md:8]:
```suggestion
---

> https://docs.github.com/en/enterprise-cloud@latest/admin/overview/about-github-for-enterprises#about-github-for-enterprises
1. [x] It allows owners to centrally manage policy and billing for multiple organizations.
```

File: content/questions/admin/question-046.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+question: "How can a GitHub organization enhance its security posture?"
+archetype: "questions"
+title: "Question 046"
+draft: false
+---
+
```
Comment (FidelusAleksander) [inline content/questions/admin/question-046.md:7]: any link to this? it's a very broad question

how about everything that GHAS covers, that surely enhances the `security posture`

File: content/questions/admin/question-049.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+question: "What can a member of a GitHub organization do by default?"
+archetype: "questions"
+title: "Question 049"
+draft: false
+---
+
+> https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization
+1. [x] They can create private repositories (if allowed by the organization's settings) and collaborate on any project within the organization.
```
Comment (FidelusAleksander) [inline content/questions/admin/question-049.md:9]: `and collaborate on any project within the organization.`

Are you sure this is true? If you are invited to an organization you still need to be added to a repository or a team that is invited to a repository.

You dont get access to all repositories by being a member

File: content/questions/admin/question-051.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+question: "How does the role of an outside collaborator differ from that of a member within a GitHub organization?"
+archetype: "questions"
+title: "Question 051"
+draft: false
+---
+
+> https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization
+1. [x] Outside collaborators are granted access to specific repositories rather than the entire organization.
```
Comment (FidelusAleksander) [inline content/questions/admin/question-051.md:9]: Same comment as i put in q49

Members dont have access to all repositories either.

File: content/questions/admin/question-062.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+---
+question: "What are the repository roles available in a GitHub organization?"
```
Comment (FidelusAleksander) [inline content/questions/admin/question-062.md:2]:
```suggestion
question: "What are the repository roles available in a GitHub organization? (Choose five.)"
```

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
I've left quite a few comments / code suggestions

Once **ALL** are addressed please re-request review so I know I can go through these again

Thanks 🚀

File: content/questions/admin/question-028.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+---
+question: "Which of the following is a management option exclusive to GitHub Enterprise Cloud?"
+archetype: "questions"
+title: "Question 028"
+draft: false
+---
+
+> https://docs.github.com/en/enterprise-cloud@latest/admin/overview/about-github-enterprise-cloud
+1. [x] Using Enterprise Managed Users to create and manage user accounts through your IdP
```
[the remark this answers is not in the document]
Comment (garysassano) [reply content/questions/admin/question-028.md:9]: It's a feature of GitHub Enterprise Cloud, but you can use GitHub Connect along with GitHub Enterprise Server.

**Review by FidelusAleksander — APPROVED** [review]
Looks good to me!

Thanks for contributing @garysassano, this makes the 4th and final practice exam finally available!

**Pull request merged**